src/api/migrations/20190704140437-create-share.js

```diff
@@ -0,0 +1,49 @@
+
+export default {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Shares', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: 'Users',
+          key: 'id'
+        },
+        onDelete: 'CASCADE',
+        onUpdate: 'CASCADE',
+        allowNull: false,
+        unique: false
+      },
+      articleSlug: {
+        type: Sequelize.STRING,
+        references: {
+          model: 'Articles',
+          key: 'slug'
+        },
```

Comment:
@MCFrank16, can you use articleId instead of articleSlug??
sequelize is complaining when we need to join tables